### PR TITLE
Dev backend cs

### PR DESCRIPTION
**Pull request by jain-anujj**

### CODE REVIEW

File: .vs/OrderingBookingModule/DesignTimeBuild/.dtbcache.v2
Comment (tu-sh-ar) [inline .vs/OrderingBookingModule/DesignTimeBuild/.dtbcache.v2:1]: Please remove bin data not related to code.

File: BuisnessLogicLayer/Services/AddressService.cs [hunk 1]
```diff
@@ -23,120 +21,69 @@ public AddressService(IUnitOfWork unitOfWork, IMapper mapper)
...
+            addNewAddress.UserId = userId;
+
             await unitOfWork.AddressRepository.AddAsync(addNewAddress);
             bool saveResult = await unitOfWork.SaveAsync();            
 
             return new ResponseDto
             {
                 Success         = saveResult,
                 StatusCode      = saveResult? 200 : 500,
-                Data            = saveResult ? addNewAddress : StringConstant.DatabaseMessage,
```
Comment (tu-sh-ar) [inline BuisnessLogicLayer/Services/AddressService.cs:37]: please make sure to return responseDTO from controller

**Review by tu-sh-ar — APPROVED** [review]
Merging code as there is dependency on other modules. Please make sure to apply all changes according to the suggestions after merge.

**Review by tu-sh-ar — APPROVED** [review]

**Review by tu-sh-ar — APPROVED** [review]

**Comment by varun-codestore** [comment]
merging commit as per the reviewed by tushar

**Pull request merged**